qupath_plus.def

```diff
@@ -16,9 +12,27 @@ From: amd64/ubuntu:jammy
          xz-utils \
          openjfx \
          unzip \
-         curl
+         curl \
+         xterm \
+         uuid-runtime
+
+# Install latest Xpra and dependencies
```

Comment:
This install is in accordance with the Xpra instructions for Deb:
https://github.com/Xpra-org/xpra/wiki/Download#-for-debian-based-distributions